frontend/lib/identification/id_card/card_content.dart

```diff
@@ -152,14 +165,34 @@ class CardContent extends StatelessWidget {
                               style: TextStyle(fontSize: 14 * scaleFactor, color: textColor),
                               textAlign: TextAlign.start,
                             ),
-                            RichText(
-                              maxLines: 1,
-                              text: TextSpan(
-                                text: "Gültig bis: ",
-                                style: TextStyle(fontSize: 12 * scaleFactor, color: textColor),
-                                children: [TextSpan(text: _formattedExpirationDate)],
+                            if (_formattedBirthday != null)
+                              Text(
+                                _formattedBirthday!,
```

Comment:
To avoid the ´!´ operator, you should initialize a local variable `formattedBirthday`, like `final formattedBirthday = _formattedBirthday;` in the context of the `build` method, and then use the variable in the null-check and below it (only then Dart can infer that it is not null; if you use a "nullable" function twice, it can't know).